blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.95-SNAPSHOT",
+  "version": "0.0.95-brand",
```

Comment:
That is the new version of the plugin. We use https://docs.npmjs.com/getting-started/using-tags to publish a "snapshot" version of the lib, so JenkinsCD can install them from the npm repo